lib/FileUploadDataSource.ts

```diff
@@ -215,7 +215,7 @@ export default class FileUploadDataSource extends RemoteGraphQLDataSource {
     const options = {
       ...request.http,
       // Apollo types are not up-to-date, make TS happy
-      body: (form as unknown) as string,
+      body: form as unknown as string,
```

Comment:
please avoid changing styling. Did eslint complained?

Reply:
yes, I can revert and disable that particular line from eslint if you prefer.